bson/src/test/unit/org/bson/codecs/pojo/PojoCustomTest.java

```diff
@@ -613,7 +613,7 @@ public void testInvalidBsonRepresentationStringEncoding() {
 
     @Test(expected = CodecConfigurationException.class)
     public void testInvalidBsonRepresentationIntDecoding() {
-        decodingShouldFail(getCodec(BsonRepresentationUnsupportedInt.class), "{'id': 'hello', s: '3'}");
```

Comment:
This failed not in decoding but in creation of the Codec. However, properties are lazy now, so the codec is only created on demand.